inference-engine/cmake/vpu_dependencies.cmake

```diff
@@ -6,14 +6,14 @@ include_guard(GLOBAL)
 
 set(VPU_SUPPORTED_FIRMWARES usb-ma2x8x pcie-ma2x8x)
 set(VPU_SUPPORTED_FIRMWARES_HASH
-    "d55a824838accec31733e4d4c45e8774bdd5690da8beefe41360f1983476e3d0"
-    "61797a77b38fc677be4cc63d730e8871bbf169686b88eabb7066b01f9d156129")
+    "0731d136349c3ad7e5de91987ee5d721f392683cbc53e3b7e7e1e200cf1eb3d0"
+    "fa97ce2fb7cc6f392af1946ae393b40c7bada4c355d38fd7ad4a92ad554b5a3e")
 
 #
 # Default packages
 #
 
-set(FIRMWARE_PACKAGE_VERSION 1714)
+set(FIRMWARE_PACKAGE_VERSION 1712)
```

Comment:
Version is being downgraded? interesting

Reply:
My bad. Will be fixed. Thanks

Reply:
fixed